### ISSUE TITLE
Dutch locale issues

### ISSUE THREAD

**Issue by jmezach** [moment/moment#3986]
It seems that the Dutch locale currently doesn't conform to how it should be used. For example, the minimized (*dd*) day names are capitalized, ie: Ma, Di, Wo, etc. These should be lower cased: ma, di, wo.

Alternatively we could use the ddd format which has lower cased names, however, these have an additional period which we sometimes need but not always. I think this should be controlled through the format string rather than being hard coded.

Basically we're expecting the following:

* dd DD-MM: di 30-05
* ddd DD-MM: di. 30-05

**Comment by stephan-v** [comment]
I have already created a pull request for this but apparently the owners of this repo are on a 2 month holiday or something.

If it takes this long for owners to respond to a pr which has been backed by the previous locale editors and passes all unit test, they should probably add owners/admins to merge pr's.

**Comment by stephan-v** [comment]
https://github.com/moment/moment/pull/3961

**Comment by icambron** [comment]
Closing in favor of #3961
